aiida/manage/configuration/schema/config-v5.schema.json

```diff
@@ -222,6 +222,11 @@
                     "type": "string",
                     "global_only": true,
                     "description": "Default user institution to use when creating new profiles."
+                },
+                "calcjob_state_tracking": {
```

Comment:
Maybe we want to put this under `logging`? So e.g. `logging.calcjob_state_tracking`.

Maybe we find also a better name (No good idea yet). Something that conveys the fact that we're tracking the 'state changes' and that we're tracking the time.

Reply:
I think I would prefer it to be separate because this is not part of the normal logging mechanism, which uses a very specific structure/design and sends the data to the daemon and/or the log table in the DB. If you still think is better to put it under logging I can change it though.

Reply:
OK - I would still think to a section for it, where in future we can add more similar things